lib/libspl/asm-generic/atomic.c

```diff
@@ -262,24 +214,14 @@ ATOMIC_SUB_NV(64, uint64_t, int64_t)
 void *
 atomic_sub_ptr_nv(volatile void *target, ssize_t bits)
 {
-	void *ptr;
-
-	VERIFY3S(pthread_mutex_lock(&atomic_lock), ==, 0);
-	ptr = (*(caddr_t *)target -= bits);
-	VERIFY3S(pthread_mutex_unlock(&atomic_lock), ==, 0);
-
-	return (ptr);
+	return __atomic_sub_fetch((void **)target, bits, __ATOMIC_SEQ_CST);
```

Comment:
cstyle: missing parenthesis around return value.

Reply:
Fixed both of those and the too-long lines